applications/FluidDynamicsApplication/python_scripts/adjoint_vmsmonolithic_solver.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
same here again

how abt multiple inheritance form `AdjointFluidSolver` AND `VMSMonolithicFluidSolver`?

This would simplify this file a lot which imo is a valid reason for multiple inheritance

Reply:
I think the long-term objective should be to skip the `AdjointFluidSolver` level so that derivation looks like `FluidSolver` -> `VMSMonolithicSolver` -> `AdjointVMSMonolithicSolver`. This would not require multiple inheritance and reduce the duplication (by skipping the extra base class, which as you say, is very similar to FluidSolver). However, we have some technical limitations (the two solvers currently have a different way of ensuring that the buffer size is properly filled).

I would propose to merge this as-is and correct the inheritance/duplication on a separate pull request, since this allows us to isolate the change in the buffer-size behavior (which has some potential for bugs).

Reply:
Sounds good

I am just suggesting, not blocking

I am saying bcs we had this problem with the chimera solver